build/cmake/options.cmake

```diff
@@ -111,7 +111,12 @@ wx_add_thirdparty_library(wxUSE_EXPAT EXPAT "use expat for XML parsing" DEFAULT_
 wx_add_thirdparty_library(wxUSE_LIBJPEG JPEG "use libjpeg (JPEG file format)")
 wx_add_thirdparty_library(wxUSE_LIBPNG PNG "use libpng (PNG image format)")
 wx_add_thirdparty_library(wxUSE_LIBTIFF TIFF "use libtiff (TIFF file format)")
-wx_add_thirdparty_library(wxUSE_NANOSVG NanoSVG "use NanoSVG for rasterizing SVG" DEFAULT builtin)
+
+# wx_add_thirdparty_library(wxUSE_LUNASVG LunaSVG "use lunasvg for rasterizing SVG (C++17 required)")
```

Comment:
Was the commented-out line kept by accident?

But I am not sure why some 3rd party libraries are added via `wx_add_thirdparty_library` and some by modifying `${wxTHIRD_PARTY_LIBRARIES}` ...

Reply:
`wx_add_thirdparty_library` adds an option with choices `off`, `sys` and `builtin`, so should only be used when the sources are distributed with wxWidgets. Otherwise you can use `wx_option` to add an option that can be turned on/off.

Some items are added manually to the `wxTHIRD_PARTY_LIBRARIES` variable, this is only for it to be listed in the CMake summary (`Which libraries should wxWidgets use?`), it doesn't set any options.